content/en/blog/falco-week-50.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Falco Weekly 50
+date: 2023-12-15
+author: Aldo Lacuku, Andrea Terzolo, Federico Di Pierro
+slug: falco-w-50
```

Comment:
If this refers to week 50 of 2023 I think we should add 2023 to the slugs so we're ready for the new year ;)

For the previous pages we can add an alias so we don't break existing links